### PR TITLE
added badge component

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

File: badge/Avatar-badge.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    .online{
+        background-color: var(--success-color);
+    }
+    .offline{
+        background-color: var(--error-color);
+    }
+    .dnd{
+        background-color: var(--grey-color);
+    }
```
Comment (prakashsakari) [inline badge/Avatar-badge.css:39]: You can clear the indentation over here by using beautify or formatter

File: badge/icon-badge.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    background-color:var(--error-color);
+    position: relative;
+    height: 1.3rem;
+    width: 1.3rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight:bolder;
+    border-radius: 50%;
+    color: white;
```
Comment (prakashsakari) [inline badge/icon-badge.css:22]: you can use a variable over here for the color

File: badge/icon-badge.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    font-weight: 500;
+     top: -2.4rem; 
+    left: 0.9rem; 
+}
+.badge-color{
+    background-color:var(---color);
```
Comment (prakashsakari) [inline badge/icon-badge.css:28]: you can rename the color variable to background-color

**Review by prakashsakari — APPROVED** [review]
Added some comments, you can have a look at it. Rest looks good to me

**Review by laxmikanthkonthum — APPROVED** [review]
looks good to me but may be declaring the lengths in a single unit through out would be nice

**Review by maitrakhatri — APPROVED** [review]
looks good

File: badge/Avatar-badge.css (hunk 1 above)
Comment (ishant005) [reply badge/Avatar-badge.css:39]: Ok i will work on it

**Pull request merged**